.travis.yml

```diff
@@ -5,8 +5,13 @@ env: # These should match the tox env list
     - TOXENV=py33
```

Comment:
Needs extra hackery for travis since they  haven't installed it yet: https://github.com/pre-commit/pre-commit/commit/2df1dc90232ad8e9985718c14025bd6e6738a52b